ci/ci/github.py

```diff
@@ -640,7 +635,7 @@ def __init__(self, index, branch, deployable, mergeable):
         self.batch_changed: bool = True
         self.state_changed: bool = True
 
-        self.n_running_batches: Optional[int] = None
+        self.n_running_batches: int = 0
```

Comment:
I think the idea was that None meant we had no information yet. For example, we maybe shouldn't run any PRs until we know how many running batches we have? I'm, OK with this change too if its simpler.